Olf2AFC.m

```diff
@@ -31,15 +31,17 @@
     TaskParameters.GUIMeta.StimDelay.Style = 'text';
     TaskParameters.GUIPanels.StimDelay = {'StimDelayAutoincrement','StimDelayMin','StimDelayMax','StimDelayIncr','StimDelayDecr','StimDelay'};
     %% FeedbackDelay
-    TaskParameters.GUI.FeedbackDelayAutoincrement = 1;
-    TaskParameters.GUIMeta.FeedbackDelayAutoincrement.Style = 'checkbox';
+    TaskParameters.GUI.FeedbackDelaySelection = 2;
+    TaskParameters.GUIMeta.FeedbackDelaySelection.Style = 'popupmenu';
+    TaskParameters.GUIMeta.FeedbackDelaySelection.String = {'Fix','FixAuto','TruncExp'};
```

Comment:
Autoincrement increases the delay gradually, and is useful for shaping long waiting times.
I wouldn't call it "FixAuto", since it's not fixed. "AutoIncr" might be a better mnemonic.